pyppeteer/browser_fetcher.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""Chromium download module."""
```

Comment:
```suggestion
"""
Chromium download module.

Chromium is being downloaded from:
https://commondatastorage.googleapis.com/chromium-browser-snapshots
see full download instructions:
https://www.chromium.org/getting-involved/download-chromium
"""
```

The chromium download instructions are awfully complicate - I'd like to keep this short docstring until we work on updating communication.

Reply:
Looks like we're actually downloading from `https://storage.googleapis.com`.